### ISSUE TITLE
Impossible to disable user tracking

### ISSUE THREAD

**Issue by roeme** [ianyh/Amethyst#376]
Connection tracking on my OS alerted me that amethyst wants to connect to ```api.mixpanel.com```.

I couldn't find any information on what data is shared, with whom, and how to disable this tracking. While I can prevent this behavior by blocking the appropriate network connections, IMHO amethyst should declare this somewhere; and/or even better, provide an option to disable it.

(In case this is "simple" crash reporting; I'm not sure why the OS-native facilities for this aren't used. In any case, even these have an option to disable.)

Apologies if this was reported before.

**Comment by ianyh** [comment]
All it does is anonymously track the number of unique launches of the app. It is intended to give me a rough idea of how many people use it. As someone who is also very privacy-conscious I agree that adding an option to disable it would be a good idea, but would kind of defeat the purpose in this case. I'll just remove it entirely for now and reintroduce it with an explicit option if I end up doing more thorough analytics.

As a note on what is shared, though: Amethyst is open source so you can search the code for instances of calls to the Mixpanel API using their Cocoa SDK (which is also open source https://github.com/mixpanel/mixpanel-iphone). Obviously, I could theoretically be introducing something at compile-time, but at some point you need to put trust in something in the chain.

**Comment by roeme** [comment]
> Amethyst is open source so you can search the code for instances 

I did, but have trouble to grok the Code on a first glance. But I think it's generally better if what's shared is declared in plain english.

> As someone who is also very privacy-conscious I agree that adding an option to disable it would be a good idea, but would kind of defeat the purpose in this case. 

An option to disable always defeats the purpose of the thing it's disabling :)

For what it's worth, I think many would be okay with sharing unique launches anonymously with you as the author, i.e.  ```ianyh.com``` (personally, I would be). The current third party is what makes me want to disable it.

Thanks for picking this up!

**Comment by ianyh** [comment]
>> Amethyst is open source so you can search the code for instances

> I did, but have trouble to grok the Code on a first glance. But I think it's generally better if what's shared is declared in plain english.

You are definitely correct. Open source does not necessarily imply complete information. I'll try to be more conscious of that in the future.

>> As someone who is also very privacy-conscious I agree that adding an option to disable it would be a good idea, but would kind of defeat the purpose in this case.

> An option to disable always defeats the purpose of the thing it's disabling :)

Well, in this case the purpose is the absolute value, not the relative value. The question "how many people use the app?" is not answerable if only some people buy in to the reporting. A question like "how many people who use this feature come back to it later?" is.
